.codecov.yml

```diff
@@ -0,0 +1,5 @@
+fixes:
+  - "^.*/site-packages/opentelemetry/sdk/::opentelemetry-sdk/src/opentelemetry/sdk/"
```

Comment:
We are still testing against site-packages, apparently we want to test the package instead of the local files according to  this PR https://github.com/open-telemetry/opentelemetry-python/pull/122

Reply:
@Oberon00 What was the reasoning behind removing the "-e" again? Don't we want to run unit tests against our locally changed files?

Reply:
@lzchen The `-e` flag was hiding issues with files that were excluded from packaging. Files were missing from the tarball / wheel files but since `-e` was being used, the tests were using the unpackaged files.

Reply:
It would be nice to have some configurable tox environment like "tox -e py37-test --installation-mode=wheel` or `tox -e px37-test --with-coverage`. But I don't think that is possible out of the box. Maybe we should generate tox.ini with some Jinja2 template 😆